python/ctsm/subset_data.py

```diff
@@ -571,9 +572,9 @@ def setup_files(args, defaults, cesmroot):
         abort("inputdata directory does not exist")
 
     # DATM data
-    datm_type = "datm_gswp3"
+    datm_type = "datm_crujra"
     dir_output_datm = "datmdata"
-    dir_input_datm = os.path.join(clmforcingindir, defaults.get(datm_type, "dir"))
+    dir_input_datm = os.path.join(TRENDYdir, defaults.get(datm_type, "dir"))
```

Comment:
@wwieder 
in finishing this PR, would you mind if I reversed the changes in subset_data and default_data, since they are coming in with #2956? Currently #2956 is next in line in the Upcoming Tags.

Reply:
Yes please do, these modifications to subset data are not required for the MIMICS work that should be the focus of the PR.  I was lazy and did them on the same branch and didn't back them out of the merge.